test/crit/Makefile

```diff
@@ -1,19 +1,29 @@
 CC ?= gcc
+GO ?= go
 
 all: test clean
 
-test: gen-imgs
+test: gen-imgs integration-test e2e-test
+
+integration-test:
+	@echo "Running integration test"
+	@./crit-test
+
+e2e-test:
```

Comment:
We need to add the corresponding dependency of targets to be able to run the tests with
`sudo make -C test/crit integration-test` and `sudo make -C test/crit e2e-test`
```diff
--- a/test/crit/Makefile
+++ b/test/crit/Makefile
@@ -3,17 +3,17 @@ GO ?= go
 
 all: test clean
 
-test: gen-imgs integration-test e2e-test
+test: integration-test e2e-test
 
-integration-test:
+integration-test: gen-imgs crit-test
        @echo "Running integration test"
        @./crit-test
 
-e2e-test:
+e2e-test: gen-imgs
        @echo "Running E2E test"
        @./crit-test.sh
 
-gen-imgs: ../loop/loop crit-test
+gen-imgs: ../loop/loop
        $(eval PID := $(shell ../loop/loop))
        criu dump -v4 -o dump.log -D ./ -t $(PID)
```